bait-files/src/main/webapp/SubmissionPage.html

```diff
@@ -0,0 +1,50 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Profile Submission Page</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="submissionPage.css">
+  </head>
+  <body>
+      <h3>What do you know about this person?</h3>
+      <p>Fill out as many fields as you can with the information that you have.</p>
+      <form action="/manual-review" method="post">
+        <ul>
+            <li>
+                <label for="name">Full Name:</label><br>
+                <input type="text" id="name" name="name">
+            </li>
+            <li>
+                <label for="username">Username(s):</label><br>
+                <input type="text" id="username" name="username">
+            </li>
+            <li>
+                <label for="email">Email:</label><br>
+                <input type="email" id="email" name="email">
+            </li>
+
+            <li>
+                <label for="address">Address:</label><br>
+                <input type="address" id="address" name="address">
+            </li>
+
+            <li>
+                <label for="phone">Phone Number:</label><br>
+                <input type="phone" id="phone" name="phone">
+            </li>
+
+            <li>
+                <label for="upload-pic">Upload Image:</label><br>
+                <input type="file" id="upload-pic" name="upload-pic">
```

Comment:
Consider only allowing for image uploads:
https://stackoverflow.com/questions/1561847/html-how-to-limit-file-upload-to-be-only-images

Reply:
good suggestion + error checking!